src/anemoi/training/config/training/default.yaml

```diff
@@ -124,3 +124,5 @@ node_loss_weights:
   _target_: anemoi.training.losses.nodeweights.GraphNodeAttribute
   target_nodes: ${graph.data}
   node_attribute: area_weight
+
+submodules_to_freeze: []
```

Comment:
This, `transfer_learning`, and future functionality I'm working in regarding fine-tuning, seem to be aligned in there purpose for model loading. I think we should consider not adding too many top-level keys but grouping them in something like `finetuning` or `model_load` or a better name where transfer learning, model loading, freezing, fine-tuning etc can live.